kuma/wiki/templates/wiki/document.html

```diff
@@ -359,11 +359,9 @@ <h2 class="offscreen">{{ _('Document Tags and Contributors') }}</h2>
                   {% include "wiki/includes/document_tag.html" %}
 
                   {% if has_contributors %}
-                    {% trans contributors_list=contributor_links(contributors) %}
                     <div class="contributors-sub">
-                      <i aria-hidden="true" class="icon-group icon-fw"></i>&nbsp;<strong>Contributors to this page:</strong> {{ contributors_list }}
+                      <i aria-hidden="true" class="icon-group icon-fw"></i>&nbsp;<strong>{{ _('Contributors to this page:') }}</strong> {{ contributors_list }}
```

Comment:
`contributors_list` was defined in the `trans` block before. You'll will need to change this to `contributor_links(contributors)` now.